examples/secured-client-with-oauth2/secured_client_with_oauth2.description

```diff
@@ -2,6 +2,8 @@
 // secured service with OAuth2 authentication. An additional `auth`
 // field is added to the HTTP client endpoint initialization in order
 // to make the simple HTTP client endpoint into a secure client endpoint.
-// Inside the `auth` field the `scheme`, `accessToken`, `clientId`,
-// `clientSecret`, `refreshToken` and `refreshUrl` is defined for
-// OAuth2 authentication.
+// Inside the `auth` field the `scheme` is defined as `http:OAUTH2` and
```

Comment:
Shall we say "In" instead of "Inside" ? Same for L6 too.

@praneesha, WDYT?

Reply:
Shall we re-word this as follows?
```suggestion
// The `scheme` of the `auth` field is defined as `http:OAUTH2` and
```

Reply:
```suggestion
// Within the `auth` field, the `scheme` is defined as `http:OAUTH2` and
```